scripts/build.js

```diff
@@ -1,76 +1,77 @@
-const fs = require('fs');
-const path = require('path');
-const util = require('util');
-
-const CleanCSS = require('clean-css');
-const { ucs2 } = require("punycode");
-const SimpleIcons = require('simple-icons');
-const svg2ttf = require('svg2ttf');
-const SVGPath = require('svgpath');
-const ttf2woff = require('ttf2woff');
-const ttf2woff2 = require('ttf2woff2');
+const fs = require('fs'),
+  path = require('path'),
+  util = require('util');
+
+const CleanCSS = require('clean-css'),
+  { ucs2 } = require('punycode'),
+  SimpleIcons = require('simple-icons'),
+  svg2ttf = require('svg2ttf'),
+  SVGPath = require('svgpath'),
+  ttf2woff = require('ttf2woff'),
+  ttf2woff2 = require('ttf2woff2');
```

Comment:
I feel like you misunderstood my comment

> I would like to see all const, let(, var?) statements with multiple values required transformed into one const-or-let-per-require. But we may leave that to a separate PR.

I actually prefer the old style (see [this comment]() for why). Do you have any particular reason for changing it into this?